agent/http.go

```diff
@@ -367,6 +367,9 @@ func (s *HTTPHandlers) wrap(handler endpoint, methods []string) http.HandlerFunc
 					continue
 				}
 				logURL = strings.Replace(logURL, token, "<hidden>", -1)
+				httpLogger.Warn("This request is using a token query parameter "+
```

Comment:
I don't foresee there ever being more than one `"token"` value but we should have this warn outside of the for loop since we wouldn't need to log more than once.